extension/src/pipeline/quickPick.ts

```diff
@@ -86,14 +90,27 @@ const verifyXFields = (xValues: QuickPickFieldValues | undefined) => {
   return x
 }
 
-const verifyYFieldsWithMultiXFields = (
-  y: PlotConfigDataAxis,
-  yValues: QuickPickFieldValues,
+const pickYFieldsWithMultiXFields = async (
+  yItems: QuickPickItemWithValue<{ file: string; field: string } | undefined>[],
   xFieldsLength: number
 ) => {
-  if (yValues.length !== xFieldsLength) {
+  const yValues = (await quickPickUserOrderedValues(
+    yItems,
+    {
+      title: `Select ${xFieldsLength} Metrics for Y` as Title
+    },
+    xFieldsLength
```

Comment:
One issue with the quick pick is that we lose separators (aka file names) when the items are collapsed:

<img width="622" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/01a923a9-7f10-4823-b4cd-3ddabc3236c4">